plugins/outputs/opentsdb/opentsdb.go

```diff
@@ -109,9 +109,12 @@ func (o *OpenTSDB) WriteHttp(metrics []telegraf.Metric, u *url.URL) error {
 		tags := cleanTags(m.Tags())
 
 		for fieldName, value := range m.Fields() {
-			metricValue, buildError := buildValue(value)
-			if buildError != nil {
-				fmt.Printf("OpenTSDB: %s\n", buildError.Error())
+			switch value.(type) {
+			case int64:
+			case uint64:
+			case float64:
+			default:
+				fmt.Printf("OpenTSDB does not support metric value: [%s] of type [%T].\n", value, value)
```

Comment:
do we really need to be logging why non-number fields aren't being sent? wouldn't an OpenTSDB user know that they can't use strings/bools?